doc/reference/reference_sql/sql-features.rst

```diff
@@ -14,443 +13,1032 @@ else we will mark it "Fail".
+                that NULLs are eliminated.   
+        *   -   E091-02   
+            -   COUNT
+            -   ``SELECT count(*) FROM t7 WHERE s1 > 0;``
+            -   :ref:`Okay <sql_aggregate_count_row>`.
+        *   -   E091-03   
+            -   MAX
+            -   ``SELECT max(s1) FROM t7 WHERE s1 > 0;``
+            -   :ref:`Okay <sql_aggregate_max>`.
+        *   -   E091-04   
+            -   MIN
+            -   ``SELECT min(s1) FROM t7 WHERE s1 > 0;``
+            -   :ref:`Okay <sql_aggregate_min>`.   
+        *   -   E091-05   
+            -   SUM
+            -   ``SELECT sum(1) FROM t7 WHERE s1 > 0;``
+            -   :ref:`Okay <sql_aggregate_sum>`.   
+        *   -   E091-06   
+            -   ALL quantifier
+            -   ``SELECT sum(ALL s1) FROM t7 WHERE s1 > 0;``   
+            -   Okay.   
+        *   -   E091-07   
+            -   DISTINCT quantifier
+            -   ``SELECT sum(DISTINCT s1) FROM t7 WHERE s1 > 0;``
+            -   Okay.  
+
+
+E101, Basic data manipulation
+-----------------------------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests
+        *   -   E101-01   
+            -   INSERT statement
+            -   ``INSERT INTO t (s1,s2) VALUES (1,''), (2,NULL), (3,55);``
+            -   :ref:`Okay <sql_insert>`.      
+        *   -   E101-03
+            -   Searched UPDATE statement
+            -   ``UPDATE t SET s1 = NULL WHERE s1 IN (SELECT s1 FROM t2);``
+            -   :ref:`Okay <sql_update>`.   
+        *   -   E101-04   
+            -   Searched DELETE statement
+            -   ``DELETE FROM t WHERE s1 IN (SELECT s1 FROM t);``
+            -   :ref:`Okay <sql_delete>`.   
+
+
+E111, Single row SELECT statement
+---------------------------------   
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests
+        *   -   E111   
+            -   Single row SELECT statement
+            -   ``SELECT count(*) FROM t;``
+            -   :ref:`Okay <sql_SELECT>`.   
+   
+   
+E121, Basic cursor support   
+--------------------------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests
+        *   -   E121-01   
+            -   DECLARE CURSOR   
+            -
+            -   Fail. Tarantool doesn't support cursors.   
+        *   -   E121-02   
+            -   ORDER BY columns need not be in select list
+            -   ``SELECT s1 FROM t ORDER BY s2;``
+            -   :ref:`Okay <sql_order_by>`.   
+        *   -   E121-03   
+            -   Value expressions in select list
```

Comment:
@pgulutzan, is this description correct? Shouldn't there be something like "Value expressions in ORDER BY clause"?

Reply:
@alexandra-mara: You are right, I erred, it should be "Value expressions in ORDER BY clause". Again, I can fix that in the English manual if there are no objections.

Reply:
@pgulutzan If you can add this change to this PR, please do, if it's not convenient I can do it

Reply:
@alexandra-mara: I believe it will be better to leave it to you, I am trying to avoid causing conflict. I see that someone has assigned me as a reviewer of this task, so please let me know when it is ready.

Reply:
@pgulutzan that was an automatic assignment, as you commented on this PR you're considered its reviewer :)